src/handlebars/installation.handlebars

```diff
@@ -64,21 +79,236 @@
               <p>and confirm the command output succeeds.</p>
             </div>
 
-            <div style="margin-top: 3.0em;">
-              <h2 class="bold">Migration Guide</h2>
-              <p>
-                If you're migrating to AdoptOpenJDK, you can learn about the differences between Oracle JDK and OpenJDK in our
-                <a href="./migration.html" target="_blank">Migration Guide</a> and any steps that you might need to take.
-                For example, how to use IcedTea-Web as an alternative to Webstart.
-              </p>
             </div>
+          \{{/each}}
+        </script>
+      </div>
+    </div>
+    <div>
+      <br><br>
+      <hr>
+    </div>
+
+    <h2 id="installers">Installers</h2>
+
+    <div class="margin-bottom">
+      <p>Installers are currently available for Windows&reg; and macOS&reg; JDK and JRE packages, which
+      can be obtained from the <a href="./index.html">Download page</a>. Installation steps are covered in the following sections:
+        <ul>
+          <li><a href="#windows-msi">Windows MSI installer packages</a></li>
+          <li><a href="#macos-pkg">macOS PKG installer packages</a></li>
+        </ul></p>
+
+    </div>
+
+    <div class="margin-bottom">
+
+      <h3 id="windows-msi">Windows MSI installer packages</h3>
+
+        <p>AdoptOpenJDK Windows installer packages are available as standard <tt>.msi</tt> files, which can be run with an interactive user interface or run silently
+        from the command line. The installer is designed for use on a per-machine basis, not per-user basis, so you can have only one installation of the MSI
+        on a machine for all users.</p>
+
+     <h4 id="windows-msi-gui">GUI installation</h4>
+
+        <p>Instructions for running an interactive installation using the Windows MSI installer.</p>
+
+        <p>1. Download the <tt>.msi</tt> file. Open it to launch the installation program.</p>
+        <p>2. Read and accept the license if you are happy with the terms.</p>
+        <p>3. On the <strong>Cutom Setup</strong> screen you can choose the features that you want to install and optionally change the default installation
+          directory. By default, AdoptOpenJDK installs to <tt>c:\Program Files\AdoptOpenJDK\<package></tt> with the following features, which you can deselect, if necessary:
```

Comment:
I think you need to use `&lt;` and `&gt;` around the package?

Reply:
Fixing